CONTRIBUTING.md

```diff
@@ -1,7 +1,11 @@
 # Contributing
 
-First, thank you for contributing to MeiliSearch! The goal of this document is to provide everything you need to start contributing to MeiliSearch.
+First, thank you for contributing to MeiliSearch!
+The goal of this document is to provide everything you need to start contributing to MeiliSearch.
```

Comment:
This is a detail do you want to have a new line at the end of this sentence. Because, the markdown format does not understand new line here, except if you add `<br>` at the end of the line

Reply:
The breakline thing is just a typo, not anything needed here.
Now about the new sentence, I think it'll be great if we add it to our other repos. Everything can be copy-pasted but the bug report link as it links to `MeiliSearch` repo.

Reply:
I let you do the change then before approving and then adding it to the other repositories